src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.swerve;
+
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+
+
+public interface SwerveModule
```

Comment:
Interfaces generally start with a capital I in their name, to denote an interface.  ISwerveModule.